@here/harp-mapview/lib/geometry/TileGeometryManager.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { MapView } from "../MapView";
+import { Tile } from "../Tile";
+import { SimpleTileGeometryLoader, TileGeometryLoader } from "./TileGeometryLoader";
+
+/**
+ * Manages the content (the geometries) of a tile. Derived classes allow different strategies that
+ * control the sequence in which the geometries if the tile are being loaded.
```

Comment:
"if the tiles" -> "of the tiles"

Reply:
Thanks, done. "I" and "O" are too close on the keyboard...